app/src/main/java/com/github/sdpteam15/polyevents/database/DatabaseInterface.kt

```diff
@@ -361,4 +361,8 @@ interface DatabaseInterface {
         uid: String = (User.currentUser as UserInterface).uid,
         userAccess: UserInterface = User.currentUser as UserInterface
     ): Observable<Boolean>
+
+    fun getItemsList(): MutableList<String>
+    fun addItem(item : String):Boolean
+    fun removeItem(item: String):Boolean
```

Comment:
It would be necessary to add asynchronous and a description for these method.
if possible homogeneously with the other method.

Reply:
I did not notice it, thank you, done in new commit